README.md

```diff
@@ -27,13 +29,67 @@ For general project information, visit the [Harmony wiki](https://wiki.earthdata
 5. [Contributing to Harmony](#Contributing-to-Harmony)
 6. [Additional Resources](#Additional-Resources)
 
+## Minimum System Requirements
+
+* A running [Docker Desktop](https://www.docker.com/products/developer-tools) or daemon instance - Used to invoke docker-based services
+* A running [Kubernetes]() cluster. [Docker Desktop](https://www.docker.com/products/docker-desktop) for Mac and Windows comes with a
+built-in Kubernetes cluster which can be enabled in preferences.
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) - A command-line application for interfacing with a Kubenetes API.
+
+## Quick-ish Start (Mac OS X / Linux)
+
+If you are interested in using a local Harmony instance to develop services, but not interested in 
+developing the Harmony code itself, the following steps are enough to start a locally running Harmony instance.
+
+1. Follow the directions for creating an Earth Data Login application and credentials in the [Earthdata Login Application Requirement](#Earthdata-Login-Application-Requirement) section below.
```

Comment:
* Earthdata Login

Also, I'd consider putting this inline here.

Reply:
Decided not to do that as it made thinks more cluttered